grafana-stack/deploy

```diff
@@ -0,0 +1,230 @@
+#!/bin/sh
```

Comment:
maybe rename this to deploy.sh?

it probably needs the boilerplate. `errexit`, `nounset`, `pipefail` probably good ideas too?